styles/signin-and-signup.css

```
@@ -0,0 +1,256 @@
+#sign-main {
+  width: 90rem;
+  height: 61.375rem;
+  padding: 14.875rem 0rem 15.75rem 0rem;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  background: var(--Linkbrary-bg, #f0f6ff);
+}
+
+.signup-main {
+  padding: 14.875rem 0rem 7.25rem 0rem;
+}
+
+.sign-container {
+  width: 25rem;
+  height: 30.75rem;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  gap: 2rem;
+}
+
+.signup-container {
+  height: 39.25rem;
+}
+
+/* 1. 상단 auth 영역 */
+.auth {
+  width: 100%;
+  height: 24.625rem;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  gap: 1.875rem;
+}
+
+.signup-auth {
+  height: 33.125rem;
+}
+
+/* 1-1. 로고 & signup link 영역 */
+.auth-links {
+  width: 13.625rem;
+  height: 4.875rem;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  gap: 1rem;
+}
+
+/* 1-1-1. 로고 */
+.auth-links-logo {
+  width: 13.16144rem;
+  height: 2.375rem;
+}
+
+/* 1-1-2. signup link */
+.auth-links-text {
+  height: 1.5rem;
+  display: flex;
+  /*
+  align-items: flex-start;
+  피그마상에서는 위와 같이 표기되어 있으나, "회원 가입하기"가 너무 붕 떠보여 center로 바꿈
+  */
+  align-items: center;
+  gap: 0.5rem;
+}
+
+.auth-links-text p {
+  color: var(--Linkbrary-Black, #000);
+  font-size: 1rem;
+  font-weight: 400;
+  line-height: 1.5rem;
+}
+
+.auth-links-text a {
+  color: var(--Linkbrary-primary-color, #6d6afe);
+  font-size: 1rem;
+  font-weight: 600;
+}
+
+/* 1-2. 이메일 & 비밀번호 영역 */
+.auth-fields {
+  width: 100%;
+  height: 12.625rem;
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  gap: 1.5rem;
+}
+
+.auth-fields-signup {
+  height: 21.125rem;
+}
+
+.auth-form {
+  width: 100%;
+  height: 5.5625rem;
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  gap: 0.75rem;
+}
+
+.auth-form-password-check {
+  height: 7rem;
+}
+
+.auth-form label {
+  height: 1.0625rem;
+  color: var(--Linkbrary-Black);
+
+  font-size: 0.875rem;
+  font-weight: 400;
+}
+
+.auth-form-input {
+  width: 100%;
+  height: 3.75rem;
+  padding: 1.125rem 0.9375rem;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-grow: 1;
+  border-radius: 0.5rem;
+  border: 1px solid var(--Linkbrary-gray20, #ccd5e3);
+  background: var(--Linkbrary-white, #fff);
+
+  color: var(--Linkbrary-gray100, #373740);
+  font-size: 1rem;
+  font-weight: 400;
+  line-height: 1.5rem;
+}
+
+.auth-form-input:focus {
+  border: 1px solid var(--Linkbrary-primary-color, #6d6afe);
+}
```

focus시 기본으로 적용되는 outline이 살아있어서 border가 가려지네요! 

```suggestion
.auth-form-input:focus {
  outline: none;
  border: 1px solid var(--Linkbrary-primary-color, #6d6afe);
}
```